pnpm-workspace.yaml

```diff
@@ -6,3 +6,21 @@ packages:
   - 'packages/presets'
   - 'packages/framework/*'
   - 'packages/affine/*'
+
+catalog:
+  '@blocksuite/icons': ^2.1.67
+  '@toeverything/theme': ^1.0.8
+  '@toeverything/y-indexeddb': 0.10.0-canary.9
+
+  '@lit-labs/preact-signals': ^1.0.2
+  '@lit/context': ^1.1.2
```

Comment:
And please also add date-fns and minimatch

Reply:
Added